### PR TITLE
chore:FPLA-2444 remove judge-notification feature toggle

### PR DESCRIPTION
**Pull request by samuelzerozero**
**Before creating a pull request make sure that:**

- [ ] PR's title includes JIRA ticket number and follow [Conventional Commits](https://www.conventionalcommits.org/) specification
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsController.java [hunk 1]
```diff
@@ -75,8 +70,6 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
     @PostMapping("/submitted")
     public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
```
Comment (samuelzerozero) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsController.java:72]: FY: That will be removed in another commit, after the case defs have been updated to prod

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsSubmittedControllerTest.java [hunk 2]
```diff
@@ -3,66 +3,23 @@
...
         super("notice-of-proceedings");
     }
 
     @Test
-    void shouldSendAllocatedJudgeNotificationWhenNoticeOfProceedingsIssuedAndEnabled()
-        throws NotificationClientException {
-        given(featureToggleService.isAllocatedJudgeNotificationEnabled(NOTICE_OF_PROCEEDINGS)).willReturn(true);
-
+    @Deprecated
+    void testSubmit() {
```
Comment (lewisbirks) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsSubmittedControllerTest.java:21]: Guessing this will be cleaned up at the same time

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java [hunk 3]
```diff
@@ -197,10 +131,6 @@ void shouldMakeCorrectCallForIsAddHearingsInPastEnabled(Boolean toggleState) {
                 (Runnable) () -> service.isCtscReportEnabled(),
                 (Runnable) () -> service.isCtscEnabled("test name"),
                 buildAttributes("report")),
-            Arguments.of(
```
Comment (lewisbirks) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java:200]: Note that we don't want to delete anymore from this method source. The testing is ensuring that the user attributes aren't persisted across requests (an issue we used to have)

**Review by lewisbirks — APPROVED** [review]
